File: 1-easy/problems/3502-minimum-cost-to-reach-every-position/solution.ts
function minCosts(cost: any): boolean | number | string | any {
  """
  :type cost: List[int]
  :rtype: List[int]
  """
  for (i in Array.from({length: 1, (cost}, (_, i) => i).length)) {
      cost[i] = min(cost[i], cost[i-1])
  return cost;
}

export { minCosts };

File: 1-easy/problems/3502-minimum-cost-to-reach-every-position/solution.py
from typing import List, Optional, Dict, Set, Tuple

class Solution:
    def minCosts(self, cost):
        """
        :type cost: List[int]
        :rtype: List[int]
        """
        for i in range(1, len(cost)):
            cost[i] = min(cost[i], cost[i-1])
        return cost
